src/components/Navbar/NavigationDrawer/OurProjectsCollapsible.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import Trans from 'next-translate/Trans';
+import useTranslation from 'next-translate/useTranslation';
+
+import NavigationDrawerItem from './NavigationDrawerItem';
+
+import Collapsible from '@/dls/Collapsible/Collapsible';
+import Link, { LinkVariant } from '@/dls/Link/Link';
+import IconArrowRight from '@/icons/arrow-right.svg';
+import IconCaretDown from '@/icons/caret-down.svg';
+import IconSquareMore from '@/icons/square-more.svg';
+import { logButtonClick, logEvent } from '@/utils/eventLogger';
+
+interface OurProjectsCollapsibleProps {
+  headerClassName?: string;
+  headerLeftClassName?: string;
+  contentClassName?: string;
+  itemTitleClassName?: string;
+  descriptionClassName?: string;
+}
+
+const OurProjectsCollapsible: React.FC<OurProjectsCollapsibleProps> = ({
+  headerClassName,
+  headerLeftClassName,
+  contentClassName,
+  itemTitleClassName,
+  descriptionClassName,
+}) => {
+  const { t } = useTranslation('common');
+
+  const onOpenChange = (isOpen: boolean) => {
+    if (isOpen) {
+      logEvent('navigation_drawer_our_projects_expanded');
+      return;
+    }
+    logEvent('navigation_drawer_our_projects_collapsed');
+  };
+
+  const projects = [
+    {
+      title: 'Quran.com',
+      href: 'https://quran.com',
+      eventName: 'navigation_drawer_project_quran_com',
+    },
+    {
+      title: 'Quran For Android',
+      href: 'https://play.google.com/store/apps/details?id=com.quran.labs.androidquran&hl=en&pli=1',
+      eventName: 'navigation_drawer_project_quran_android',
+    },
+    {
+      title: 'Quran iOS',
+      href: 'https://apps.apple.com/us/app/quran-by-quran-com-%D9%82%D8%B1%D8%A2%D9%86/id1118663303',
+      eventName: 'navigation_drawer_project_quran_ios',
+    },
+    {
+      title: 'QuranReflect.com',
+      href: 'https://quranreflect.com/',
+      eventName: 'navigation_drawer_project_quran_reflect',
+    },
+    {
+      title: 'Sunnah.com',
+      href: 'https://sunnah.com/',
+      eventName: 'navigation_drawer_project_sunnah',
+    },
+    {
+      title: 'Nuqayah.com',
+      href: 'https://nuqayah.com/',
+      eventName: 'navigation_drawer_project_nuqayah',
+    },
+    {
+      title: 'Legacy.quran.com',
+      href: 'https://legacy.quran.com',
+      eventName: 'navigation_drawer_project_legacy',
+    },
+    {
+      title: 'Corpus.quran.com',
+      href: 'https://corpus.quran.com',
+      eventName: 'navigation_drawer_project_corpus',
+    },
+  ];
+
+  const handleProjectClick = (eventName: string) => () => {
+    logButtonClick(eventName);
+  };
+
+  return (
+    <Collapsible
```

Comment:
Clicking on the Collapsible tip does not take any effect, imo, we should make the entire item clickable and toggles the collapsible.

![Image](https://github.com/user-attachments/assets/f1eaa817-c9ab-4e81-adc5-cf5c1915d9fc)